### ISSUE TITLE
model: flat TARGET_LOSS_RATIO across all tower layers ignores layer-specific market pricing

### ISSUE THREAD

**Issue by AlexFiliakov** [AlexFiliakov/Ergodic-Insurance-Limits#1587]
## Symptom (notebook `07_hjb_insurance_optimization.ipynb`, cell 10)

`make_layer_pricers()` and `analytical_layer_premium()` apply a single `TARGET_LOSS_RATIO = 0.65` to every layer in the 4-layer tower:

```python
def analytical_layer_premium(attachment, limit, pricers=None):
    expected_loss = sum(p.expected_layer_loss(attachment, limit) for p in pricers)
    return expected_loss / TARGET_LOSS_RATIO  # ← same denominator for all layers
```

Cell 16's ROL validation table shows the *consequence*: layer ROLs decrease monotonically from 37.6% (primary at $10K attach) to 0.7% (cat at $50M attach) — but only because expected loss decreases. The *loading multiple* (premium / E[loss]) is identical at 1.54× for every layer.

This contradicts real P&C market pricing, where:
- **Working layer (≤$5M)**: heavy loading, 1.5-3× E[loss] (capital cost, frictional costs, profit)
- **First excess ($5-25M)**: moderate loading, 1.2-1.8× E[loss]
- **Second excess ($25-50M)**: thin loading, 1.0-1.3× E[loss]
- **Cat ($50-100M)**: very thin loading, often 1.0× E[loss] or below in soft markets (insurer accepts the negative-EV trade for ALAE/reinsurance arbitrage and capacity placement)

By applying 1.54× uniformly, the notebook *overprices* the cat layer (where the variance benefit is largest) and *underprices* the working layer (where the operational nuisance value is largest). Both errors push toward "skip insurance."

## Evidence

Cell 16 ROL table:
```
       SIR       Premium  E[L_ret]/event   E[L_ret]/yr@ref
$     10,000  $  1,634,718  $        7,355  $         31,259
$  4,000,000  $    146,696  $      262,244  $      1,114,539
```

`make_layer_pricers` (cell 10) returns one `LayerPricer` per loss component (attritional/large/cat) but they're aggregated through a single `TARGET_LOSS_RATIO`.

## Why this matters

The HJB optimizes against a price model that doesn't match the market it claims to describe. A user calibrating the notebook to a real client's tower would find that real premiums for the cat layer are 10-30% of the notebook's prediction. Under those real premiums, the HJB would likely recommend buying *more* high-attachment coverage than under the flat 1.54×.

This is also a partial root cause of #issue-7: the cat layer overpricing inflates total premium, which makes "buy nothing" beat "buy everything" by a larger margin than real markets would allow.

## Remediation options

1. **(Recommended) Layer-specific `TARGET_LOSS_RATIO`.** Replace the scalar with a dict or list keyed to layer index:
   ```python
   TARGET_LOSS_RATIO = {
       'primary':  0.55,  # 1.82x loading
       'excess_1': 0.65,  # 1.54x
       'excess_2': 0.78,  # 1.28x
       'cat':      0.95,  # 1.05x
   }
   ```
   Threading these through `analytical_layer_premium` and `make_program` is mechanical. Tradeoff: introduces 4 new free parameters; need to document defaults.

2. **Use ILF (Increased Limit Factor) curves.** Industry-standard pricing for higher layers: `premium(a, l) = base_rate(a) * limit * ILF(a+l) / ILF(a)`, with `ILF` calibrated to ISO/RAA standards. Tradeoff: more code; requires ILF data.

3. **Add a "loading curve" parameter `loading(attach)` returning a per-layer multiplier**, default linearly decreasing from 1.8 at attach=$0 to 1.05 at attach=$50M. Tradeoff: less industry-standard but easier to explain.

4. **Keep the flat loading, document the limitation** in the cell-10 markdown. Tradeoff: pedagogically clean but quantitatively wrong; downstream calibration work breaks.

## Recommendation

Land option 1 with sensible defaults (e.g., `{0.55, 0.65, 0.75, 0.95}`) — this is the minimal change with the maximal realism gain. Combine with #issue-7 (overall LR calibration): the two together restore an interior HJB optimum without needing to fundamentally change the loss model. Option 2 (ILF curves) is the right long-term direction but is out of scope for this fix.

**Comment by AlexFiliakov** [comment]
**Resolved (core); distortion-functional refinement split to #1601 -- closing.**

Implemented on `fix/nb07-layer-loadings-regret-refinement` (cell 10). The flat `premium = E[layer loss] / TARGET_LOSS_RATIO` is replaced by the per-layer SD principle this issue proposed:

`premium(a,w) = E[L_a] * (1 + theta * CV(L_a))`, where
- `CV(L_a)` is the **annual-aggregate** compound-Poisson layer-loss CV, `sqrt(sum_c lambda_c * E[Y_c^2]) / (sum_c lambda_c * E[Y_c])`, computed from new closed-form **limited second moments** `E[min(X,L)^2]` for the lognormal and Pareto severities (validated vs Monte Carlo to <1%);
- **theta is back-derived once from the primary working layer** so the primary keeps the headline `TARGET_LOSS_RATIO`, then held static across the tower (a fixed market schedule at `REFERENCE_REVENUE`) -- exactly the remediation in the issue. theta lands in the 0.1-0.5 range flagged here.

New cell-10 API: `_lev2`, `_layer_second_moment`, `layer_loss_cv`, `layer_loading_multiple`, `THETA_RISK_LOADING`. The validation table now reports per-layer `CV / loading / implied-LR` plus a 4-layer-tower table, replacing the flat 1.54x/1.43x the issue pointed at -- loadings now rise with attachment (primary at `1/LR`, excess progressively higher).

**Heavy-tail caveat (this issue's "Future enhancement"):** the raw SD multiple explodes for the remote cat layer (Pareto alpha=2.05, where sigma is a poor tail summary). I bounded it with a documented per-layer **loss-ratio floor** (`MIN_LAYER_LOSS_RATIO`, which caps the loading) rather than adopting a Wang/PH distortion functional now. That distortion-functional refinement is split into follow-up #1601 so it isn't lost.

Closing the per-layer-loading work as complete.
